README.md

```diff
@@ -230,6 +230,32 @@ You can install it with `go install github.com/vektra/mockery/v2@latest`.
 > **Note**  
 > Be sure to add the `mockery` binary to your `$PATH` environment variable so that `make` can find it.
 
+
+## Docker
+
+To aim in testing & developing trento, we have added a [Dockerfile](Dockerfile) that can be
+used to build a container image to run `trento` on and a [docker-compose.yml](docker-compose.yml) file
+that allows to pull all the dependencies that trento needs and run them in containers
+too.
+
+To build the image:
+```shell
+git clone https://github.com/trento-project/trento.git
+cd trento
+docker build -t trento ./
+```
+
+The only parameter that needs to be defined is 
```

Comment:
@rtorrero Is there something missing here? It looks like the phrase is not finished

Reply:
Yes! Thanks for pointing that out, I've removed it as what I wanted to put there no longer makes sense